packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -697,6 +698,143 @@ describe("handleNewBooking", () => {
+              eventTypes: [
+                {
+                  id: 1,
+                  slotInterval: 30,
+                  length: 30,
+                  minimumReschedulingNotice: 120, //2 hours
+                  users: [
+                    {
+                      id: 101,
+                    },
+                  ],
+                },
+              ],
+              bookings: [
+                {
+                  uid: uidOfBookingToBeRescheduled,
+                  eventTypeId: 1,
+                  status: BookingStatus.ACCEPTED,
+                  startTime: `${plus1DateString}T05:00:00.000Z`,
+                  endTime: `${plus1DateString}T05:30:00.000Z`,
+                  metadata: {
+                    videoCallUrl: "https://existing-daily-video-call-url.example.com",
+                  },
+                  references: [
+                    {
+                      type: appStoreMetadata.dailyvideo.type,
+                      uid: "MOCK_ID",
+                      meetingId: "MOCK_ID",
+                      meetingPassword: "MOCK_PASS",
+                      meetingUrl: "http://mock-dailyvideo.example.com",
+                    },
+                    {
+                      type: appStoreMetadata.googlecalendar.type,
+                      uid: "ORIGINAL_BOOKING_UID",
+                      meetingId: "ORIGINAL_MEETING_ID",
+                      meetingPassword: "ORIGINAL_MEETING_PASSWORD",
+                      meetingUrl: "https://ORIGINAL_MEETING_URL",
+                      externalCalendarId: "existing-event-type@example.com",
+                      credentialId: undefined,
+                    },
+                  ],
+                },
+              ],
+              organizer,
+              apps: [TestData.apps["google-calendar"], TestData.apps["daily-video"]],
+            })
+          );
+
+          const videoMock = mockSuccessfulVideoMeetingCreation({
+            metadataLookupKey: "dailyvideo",
+          });
+
+          const calendarMock = mockCalendarToHaveNoBusySlots("googlecalendar", {
+            create: {
+              uid: "MOCK_ID",
+            },
+            update: {
+              uid: "UPDATED_MOCK_ID",
+              iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
+            },
+          });
+
+          const mockBookingData = getMockRequestDataForBooking({
+            data: {
+              eventTypeId: 1,
+              rescheduleUid: uidOfBookingToBeRescheduled,
+              responses: {
+                email: booker.email,
+                name: booker.name,
+                location: { optionValue: "", value: BookingLocations.CalVideo },
+              },
+            },
+          });
+
+          //minimum notice is 2 hours, but we are trying to reschedule with 1 hour remaining
+          const mockFailingCurrentTime = new Date(dayjs(`${plus1DateString}T03:00:00.000Z`));
+          vi.useFakeTimers();
+          vi.setSystemTime(mockFailingCurrentTime);
```

Comment:
Easier and more readable than mocking dayjs().utc(), changes system time so that while getting currentTime in validateRescheduleWindows.ts we get the desired time.